src/index.jsx

```diff
@@ -33,52 +38,134 @@ class App extends Component {
 
     if (experienceId === NaN) {
       experienceId = 1;
-    };
+    }
 
     axios.get(`/photos/${experienceId}`)
       .then((res) => {
-
+        const photos = res.data;
+        const heroPhoto = photos[0];
         this.setState({
-          photos: res.data,
+          photos: photos,
+          heroPhoto: heroPhoto,
         });
-
+        console.log('HERO PHOTO: ', this.state.heroPhoto);
       })
       .catch((err) => {
         console.log('Axios Error: ', err);
       });
   }
 
+  openModal() {
+    let openOrClose= !!(true);
+    this.setState({
+      isOpen: openOrClose,
+    });
+  }
+
+  //conditional render
+    //if isOpen is false show regular
+    //if true, just show modal
+
+  onClose() {
+    this.setState({
+      isOpen: openOrClose,
+    });
+  }
+
+  swipeToLast() {
+    const newIndex = this.state.heroPhotoIndex - 1;
+
+    const newHeroPhoto = this.state.photos[newIndex];
+
+    this.setState({
+      heroPhotoIndex: newIndex,
+      heroPhoto: newHeroPhoto,
+    });
+    console.log('NEW INDEX: ', newIndex);
+  }
+
+  swipeToNext() {
+    const currentPhotoIndex = this.state.heroPhotoIndex + 1;
+    const lastHeroPhoto = this.state.photos[currentPhotoIndex];
+    this.setState({
+      heroPhotoIndex: currentPhotoIndex,
+      heroPhoto: lastHeroPhoto,
+    });
+  }
 
   render() {
```

Comment:
I'm curious to see if this conditional render works! (I would have just done some 'if' conditionals)